string move
---
string fen_string
string game_state